include/wlr/util/box.h

```diff
@@ -0,0 +1,59 @@
+/*
+* This is a stable interface of wlroots. Future changes will be limited to:
+ *
+ * - New functions
+ * - New struct members
+ * - New enum members
+ *
+ * Note that wlroots does not make an ABI compatibility promise - in the future,
+ * the layout and size of structs used by wlroots may change, requiring code
+ * depending on this header to be recompiled (but not edited).
+ *
+ * Breaking changes are announced by email and follow a 1-year deprecation
+ * schedule. Send an email to ~sircmpwn/wlroots-announce+subscribe@lists.sr.ht
+ * to receive these announcements.
+ */
+#ifndef WLR_UTIL_WLR_BOX_H
```

Comment:
This should be named `WLR_UTIL_BOX_H`: we derive the define name from the file path.